x-pack/elastic-agent/pkg/agent/cmd/common.go

```diff
@@ -19,7 +24,11 @@ import (
 	"github.com/elastic/beats/v7/x-pack/elastic-agent/pkg/cli"
 )
 
-const defaultConfig = "elastic-agent.yml"
+const (
+	defaultConfig = "elastic-agent.yml"
+	hashLen       = 6
+	commitFile    = ".elastic-agent.active.commit"
```

Comment:
how is this different from the build hash? Will it change during upgrade?

Reply:
the change is in location, build hash file we have is in different location than our configuration files and differs based on platform/type of package.
i introduced this hidden file so it sits right next to configuration and if needed can change format